daemon/rpc/handlers.go

```diff
@@ -157,9 +179,9 @@ func (server *Server) SwapOut(ctx context.Context, req *SwapOutRequest) (*SwapOu
 		DestinationChain:   models.Bitcoin,
 		ClaimPubkey:        hex.EncodeToString(claimKey.Serialize()), // TODO: Add claim pubkey to the model
 		PaymentRequest:     swap.Invoice,
-		AmountSats:         int64(amount),     // nolint:gosec
-		ServiceFeeSats:     int64(serviceFee), // nolint:gosec
-		MaxRoutingFeeRatio: 0.005,             // 0.5% is a good max value for Lightning Network - TODO: pass this as a parameter
+		AmountSats:         int64(amount), // nolint:gosec
+		ServiceFeeSats:     serviceFeeSats.IntPart(),
+		MaxRoutingFeeRatio: 0.005, // 0.5% is a good max value for Lightning Network - TODO: pass this as a parameter
```

Comment:
I wonder if this needs better tuning as this is way sensitive for 40swap 🤔

Reply:
40swap could have their own max routing fee for swap ins. shouldn't affect this one, right?

Reply:
Sorry I mean `40swapd` end users, I do think that the max routing fee should be a parameter with a default value but shall be configurable, its a big chunk % of the cost for swaps sometimes

Reply:
makes sense. Will do in a following PR